### PR TITLE
Testing To Do list

### PR DESCRIPTION
**Pull request by LamiaSristy**
Added Unit Test Suits for the JS files.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team. 👋

**You both did pretty well here. 👌👌**

However, let's make a few fixes.

- [ ] Please check comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/__test__/helper.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { getDataFromLocalStorage, updateLocalStorage } = require('../helper');
```
Comment (mikenath223) [inline src/__test__/helper.test.js]: - [ ] Please change this to es6 imports. Fix this and other affected modules.

**Comment by mikenath223** [comment]
- [ ] You have a `module.export` in tour helper. Please change to es6 `export`. Kindly do this for other affected modules too.

File: src/__test__/helper.test.js (hunk 1 above)
Comment (talhawaqar) [reply src/__test__/helper.test.js]: This is not mendatory.

Comment (talhawaqar) [reply src/__test__/helper.test.js]: This project is already approved only tests need to be checked.

Comment (mikenath223) [reply src/__test__/helper.test.js]: Kindly note that using es6 syntax is mandatory @talhawaqar

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @LamiaSristy and @talhawaqar :wave: 

### I think you still didn't make the requested changes from the previous TSE, please make them:

- [x] Use ES6 syntax you learned from the previous lesson, you need to use `import` instead of `require` when using other modules ([screenshot from Pathwright](https://user-images.githubusercontent.com/59691171/102891914-d1e3e780-443d-11eb-8309-67d0b2c0d0e1.png)).

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello again @LamiaSristy and @talhawaqar :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**